### ISSUE TITLE
Allow for header section of landing page to be video

### ISSUE THREAD

**Issue by bedeho** [Joystream/joystream#1425]
This will perhaps also require a volume control/mute button. We simply pick the best exemplar @blrhc finds, perhaps we can even make it Schism, as that is funded by Jsgenesis.

**Comment by bedeho** [comment]
@blrhc will provide the video here, and this should be done before I do the UI review.

**Comment by kdembler** [comment]
Superseded by #1627
